### ISSUE TITLE
General Improvements

### ISSUE THREAD

**Issue by evertonlperes** [rancher-sandbox/rancher-desktop#1065]
General improvements and opportunities after moved from Spectron to Playwrigth:

- [ ] Clobber RD Settings instead having it write/read from disk (rely on `process.env.NODE_ENV=test`).
- [x] Improve `main.e2e.spec.ts`, including more assertions and validations. Blocked by #1317 
- [ ] Bypass networking root access password input on `TEST` mode (dockerd only). - Could be fixed by #1224 
- [ ] Alternative way to detect if the app was loaded properly ([don't rely](https://github.com/rancher-sandbox/rancher-desktop/pull/1146#discussion_r773442249) on `.progress` be detached or hidden). - Blocked by #1217 
- [x] Improve TestUtils with more global methods.

**Comment by IsaSih** [comment]
Any e2e topics are already done. Remaining issues related to enhancements only
